05/app.js

```diff
@@ -9,6 +9,36 @@ const stats = {
 
 /* tutaj umieść swój kod */
 
+const pList = document.querySelectorAll('p');
+const aList = document.querySelectorAll('a');
+let counter=0;
+
+
+function updateDisplay(counter) { //wydaje mi sie ze to mozna zrobić lepiej, próbowałam z dodatkową f-cja do zmiany danych w obiekcie ale nietety, moze mi podpowiesz gdzie robie blad
+    stats.paragraphs.p1 = counter;
+}
+
+pList.forEach(function(element){
+    element.addEventListener('click', function(e){
+        if(e.target.dataset.id === "p1"){
+            counter ++;
+            updateDisplay(counter);
+        }
```

Comment:
Tutaj powinnaś sprawdzić jaki element jest klikany np. `e.target.tagName === 'P'` - czy jest to paragraf, jeśli tak to wtedy można iść dalej i np.
```
const id = e.target.dataset.id
if(!stats.paragraphs[id]) { // jeśli nie istnieje to tworzymy nowa wartość
    stats.paragraphs[id] = 0
}

stats.paragraphs[id]++; // zwiększamy o 1 wartość
```

Daj znać czy taka podpowiedź wystarczy ;)

Reply:
Hej, odp wystarczyła, teraz wszystko jest takie logiczne :) dzięki